users/models.py

```diff
@@ -40,6 +46,11 @@ def create_user(username, email, password, first_name, last_name, role, team):
         else:
             raise Exception("Error creating user")
 
+    @staticmethod
+    def checkValidName(string):
```

Comment:
the pythonic way is like:
```suggestion
    def check_valid_name(string):
```